### PR TITLE
FIX: HasSettingsField now adheres to $connection override on model (#62)

### PR DESCRIPTION
**Pull request by bickle66**
Modified the HasSettingsField.php to allow for reading the model's connection value (default or otherwise) as proposed on issue #62 
Updated readme.md to document change

### CODE REVIEW

File: src/Traits/HasSettingsField.php [hunk 1]
```diff
@@ -74,6 +74,14 @@ public function getSettingsFieldName(): string
         return $this->settingsFieldName ?? config('model_settings.settings_field_name');
     }
 
+    /**
+     * @return string
+     */
+    public function getConnectionName(): string
```
Comment (ametad) [inline src/Traits/HasSettingsField.php:80]: This function already is part of Laravel, so why is this needed here?

Comment (ametad) [reply src/Traits/HasSettingsField.php:80]: I see now! `Scheme::connection()` needs a value and `null` is not valid. So there must be a fallback value.

**Comment by ametad** [comment]
@glorand Could you please merge this pull request? It would resolve the problem discussed in #43 for me.

**Comment by glorand** [comment]
> @glorand Could you please merge this pull request? It would resolve the problem discussed in #43 for me.

Tomorow will take a closer look, and probably will be merged

**Comment by ametad** [comment]
Thank you, great!

**Comment by ametad** [comment]
My problems are history now, yes!
